lambda-functions/Lambda/psycopg2-no-SQLAlchemy/lambda-reddit-streaming.py

```diff
@@ -0,0 +1,57 @@
+import pandas as pd
+import praw
+from access_psycopg2 import make_conn,fetch
+
+# thanks Mac for the Reddit credentials
+
+REDDIT_CLIENT_ID = "VXYCrQAymaBrLQ"
+REDDIT_CLIENT_SECRET = "0Wwvv9zptchx7iqweK52cvbtX4A"
+REDDIT_USER_AGENT = "WSB"
+
+reddit = praw.Reddit(client_id=REDDIT_CLIENT_ID,
+                     client_secret = REDDIT_CLIENT_SECRET,
+                     user_agent=REDDIT_USER_AGENT)
+
+def subreddit_query(subreddits):
```

Comment:
This seems to be a duplication of [lambda_reddit_streaming.py](lambda-functions/Lambda/SQLAlchemy/lambda_reddit_streaming.py) perhaps delete this file, or reduce the duplication?